### ISSUE TITLE
zmq_msg_t changed in ZeroMQ 4.1

### ISSUE THREAD

**Issue by orivej** [rpav/ZMQ4L#1]
JFYI, `zmq_msg_t` size [has increased](http://zeromq.org/docs:4-1-upgrade), which will cause corruption if ZMQ4L is used with ZeroMQ 4.1 with the current spec files.

Originally reported by @sbalousek at https://github.com/orivej/pzmq/pull/10.

I do not use this library, but I would like to know how to best handle such a change with cl-autowrap, preserving backward compatibility. One option is to simply regenerate the specs, since allocating more for `zmq_msg_t` should continue to work with older versions of ZeroMQ .

**Comment by rpav** [comment]
Yeah I'll likely just regen the specs myself... I don't really use this at the moment either and havne't been following ZMQ.

I'm not sure there's a particularly amazing solution either way.  Ideally with autowrap you'd just keep multiple sets of specs and maybe have some specific loading by-version.  This kinda sucks with the size of .spec files, but it's the Most Correct(tm) solution.  I'd like to get autowrap "v2" done before too long, and only use .spec files temporarily to produce minimal necessary data, which should help that.
